content/docs/learning-guide/tutorials/environments/create-terrain-from-images.md

```diff
@@ -0,0 +1,314 @@
+
+* A shape component. Most often, this is a **Shape Reference** component that references the **Axis Aligned Box Shape** component in the Terrain Spawner entity.
+
+#### Add a Terrain Surface Gradient List
+
+These steps associate the gradient entities with the surface tag names you created.
+
+1. With the Terrain Spawner selected, add a **Terrain Surface Gradient List** component.
+
+2. In the **Terrain Surface Gradient List** component, click the {{< icon "add.svg" >}} **Add** button to add a new gradient slot.
+
+3. To the right of the **Gradient Entity** property, click the {{< icon "picker.svg" >}} **Picker** button, then click a gradient entity in Entity Outliner to reference it.
+
+4. For the **Surface Tag** property, select the appropriate surface tag name from the list. Note that the list contains the surface tag names you created in Asset Editor.
+
+5. Repeat steps 4 though 6 for each detail material you want to apply to the surface.
+
+#### Add a Terrain Surface Materials List
+
+This final task assigns the detail materials to the terrain based on the surface tag names and associated gradient entities.
+
+1. With the Terrain Spawner selected, add a **Terrain Surface Materials List** component.
+
+2. To the right of the **Default Material** property, click the {{< icon "file-folder.svg" >}} **File** button and select your default material asset.
+
+3. To the right of the **Material Mappings** property group, click the {{< icon "add.svg" >}} **Add** button to add a new material slot.
+
+4. In the **Surface Tag** property, select one of the surface name tags you created.
+
+5. To the right of the **Material asset** property, click the {{< icon "file-folder.svg" >}} **File** button and select the material asset to associate with the surface tag.
+
+6. Repeat steps 10 through 12 for each remaining detail material you want to apply to the terrain.
+
+As you add detail materials and surface tag name pairs to the **Terrain Surface Materials List** component, the materials appear on the terrain surface.
+
+## Add terrain physics
+
+You can add collision to terrain for PhysX simulation and apply [physics materials](/docs/user-guide/interactivity/physics/nvidia-physx/materials/) that define the physical properties of the terrain such as dynamic and static friction. To add physics to a terrain, do the following:
+
+1. Add a **PhysX Heightfield Collider** component to the **Terrain Spawner** entity.
+
+2. The **PhysX Heightfield Collider** component displays a warning about missing a required component. Choose **Add Required Component** and select **Terrain Physics Heightfiled Collider** from the list.
+
+3. The **PhysX Heightfield Collider** component adds a collider to the terrain so that dynamic PhysX objects collide with it. You can enable the **Use Baked Heightfield** property for better performance if the heightfield doesn't change at runtime, such as when the terrain in generated from an image gradient (heightmap).
```

Comment:
```suggestion
3. The **PhysX Heightfield Collider** component adds a collider to the terrain so that dynamic PhysX objects collide with it. You can enable the **Use Baked Heightfield** property for better performance if the heightfield doesn't change at runtime, such as when the terrain is generated from an image gradient (heightmap).
```